auth/common_test.go

```diff
@@ -196,20 +196,20 @@ func testAudit(
 
 	for _, c := range tests {
 		t.Run(c.user, func(t *testing.T) {
-			req := require.New(t)
+			r := require.New(t)
 
-			db, err := dsql.Open("mysql", connString(c.user, ""))
-			req.NoError(err)
-			_, err = db.Query(c.query)
+			db, e := dsql.Open("mysql", connString(c.user, ""))
```

Comment:
why `e`?

Reply:
To shut up `go vet` about reusing `err` in the enclosing context.

Reply:
`var db *dsql.DB` actually. It's one line more, but ok, `err` is certainly more easy to human-parse than just `e`.